cmd/get/launch_plan.go

```diff
@@ -41,10 +42,22 @@ Retrieves particular version of launchplan by name within project and domain.
 
  flytectl get launchplan -p flytesnacks -d development  core.basic.lp.go_greet --version v2
 
-Retrieves launchplan by filters.
+Retrieves all the launch plans with filters.
 ::
-
- Not yet implemented
+ 
+  bin/flytectl get launchplan -p flytesnacks -d development --filter.field-selector="name=core.basic.lp.go_greet"
+ 
+Retrieves specific launch plans with filters.
```

Comment:
By specific does this mean with entity search across all versions and if yes please reword it to signify that.